tools/README.md

```diff
@@ -24,11 +24,18 @@ git clone --recurse-submodules https://github.com/commaai/openpilot.git
 
 **2. Run the setup script**
 
+Ubuntu 24.04:
 ``` bash
 cd openpilot
 tools/ubuntu_setup.sh
 ```
 
+macOS:
+``` bash
+cd openpilot
+tools/mac_setup.sh
```

Comment:
This is great, but I believe it can be left as a single OS-agnostic block if you use the `op.sh` script

```suggestion
tools/op.sh setup
```